code/game/jobs/whitelist.dm

```diff
@@ -1,15 +1,20 @@
 #define WHITELISTFILE "data/whitelist.txt"
-
+	
 var/list/whitelist = list()
+var/list/alien_whitelist = list()
 
-/hook/startup/proc/loadWhitelist()
+// /proc/load_popularity_contest_unlockables()

```

Comment:
Damn, you didn't go through with my proc name...

Reply:
I left the comment in though